CustomDimensions.php

```diff
@@ -280,10 +281,13 @@ public function addConversionInformation(&$conversion, $visitInformation, Tracke
 
         foreach ($dimensions as $dimension) {
             $index = (int) $dimension['index'];
-            if ($dimension['scope'] === self::SCOPE_VISIT && in_array($index, $conversionIndexes)) {
+            if (in_array($dimension['scope'], array(self::SCOPE_ACTION, self::SCOPE_VISIT)) && in_array($index, $conversionIndexes)) {
                 $field = LogTable::buildCustomDimensionColumnName($dimension);
 
-                if (array_key_exists($field, $visitInformation)) {
+                if ($dimension['scope'] === self::SCOPE_ACTION && array_key_exists($field, $actionDimensions)) {
```

Comment:
Have just tested this, but in the case of manual conversions the action is set to 'null'. I guess this is again due to the check in GoalsRequestProcessor::processRequestParams. Any suggestions?